components/toggle_modal_button_redux/toggle_modal_button_redux.jsx

```diff
@@ -56,11 +61,14 @@ export default class ModalToggleButtonRedux extends React.Component {
             };
         }
 
+
         return (
             <button
                 {...props}
                 className={'style--none ' + props.className}
+                data-toggle='modal toggle'
                 onClick={clickHandler}
+                aria-label={children + this.context.intl.formatMessage({id: 'accessibility.button.dialog', defaultMessage: ' Dialog'})}
```

Comment:
@asaadmahmood `children` here can be a set of components and should not be treated as a string